src/Vimeo/Vimeo.php

```diff
@@ -347,7 +347,11 @@ public function replace($video_uri, $file_path, $upgrade_to_1080 = false, $machi
     private function perform_upload($file_path, $ticket)
     {
         if ($ticket['status'] != 201) {
-            throw new VimeoUploadException('Unable to get an upload ticket.');
+            if(isset($ticket['body']['error'])){
+		$ticket_error = "[" . $ticket['body']['error'] . "]";
+	    }
+
+ 	    throw new VimeoUploadException('Unable to get an upload ticket.' . $ticket_error);
```

Comment:
I don't think undefined is a thing in PHP, isset or empty should both work fine!